src/main/scala/ai/eto/rikai/sql/execution/ShowModelsCommand.scala

```diff
@@ -6,10 +6,20 @@ import ai.eto.rikai.sql.catalog.{MLCatalog, Model}
 import org.apache.spark.sql.catalyst.expressions.{Attribute, AttributeReference}
 import org.apache.spark.sql.types.StringType
 
+
+import org.json4s._
+import org.json4s.jackson.Serialization
+import org.json4s.jackson.Serialization.write
+
+
 case class ShowModelsCommand() extends RunnableCommand {
+
+    implicit val formats = Serialization.formats(NoTypeHints)
+
     override val output: Seq[Attribute] = Seq(
         AttributeReference("Model Name", StringType, nullable = false)(),
-        AttributeReference("Model Path", StringType, nullable = false)()
+        AttributeReference("Model Path", StringType, nullable = false)(),
```

Comment:
We can just use `Name, Path, Options`  fo the name?

Are `path` and `options` nullable?

Reply:
In the `master` branch, we introduced a [`SparkRunnable`](https://github.com/eto-ai/rikai/blob/master/src/main/scala/ai/eto/rikai/sql/spark/SparkRunnable.scala), we might want to only display `SparkRunnable`s in ```SHOW MODELS```?